pdu/pdu.py

```diff
@@ -167,26 +186,27 @@ def do_outlet_power_control(self, outlets: list):
         Returns:
             None
         """
+        fname = 'do_outlet_power_control'
         for outlet in outlets:
             retries = 0
             while True:
                 if retries >= 5:
-                    print('do_outlet_power_control: %s exceeded retries for ' \
-                            'outlet %s at %s, giving up' %
-                            (outlet['operation'], outlet['name'], self._host))
+                    self.error(f'{fname}: {outlet["operation"]} exceeded ' \
+                        'retries for outlet ' \
+                        '{outlet["name"]} at {self._host}, giving up')
```

Comment:
I think this needs an f on the last string
i.e.
```
self.error(f'{fname}: {outlet["operation"]} exceeded ' \
         'retries for outlet ' \
         f'{outlet["name"]} at {self._host}, giving up')
```
And if this is right, other places in this code need the same addition.